integrations/fullstory/lib/index.js

```diff
@@ -73,12 +76,13 @@ FullStory.prototype.identify = function(identify) {
 
   var newTraits = foldl(
     function(results, value, key) {
+      var rs = results;
```

Comment:
Just curious, why do we need this?

Reply:
it was to get around the eslint no-param-reassign rule.